contrib/gn_module_occhab/backend/gn_module_occhab/migrations/167d69b42d25_import.py

```diff
@@ -1109,18 +1110,6 @@ def upgrade():
         column_name="id_nomenclature_collection_technique",
         server_default=sa.func.pr_occhab.get_default_nomenclature_value("TECHNIQUE_COLLECT_HAB"),
     )
-    op.alter_column(
-        schema="pr_occhab",
-        table_name="t_stations",
-        column_name="unique_id_sinp_station",
-        nullable=False,
```

Comment:
En fait les colonnes étaient déjà `NOT NULL` à l’origine, on peut donc se contenter de tous virer.